sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -114,7 +114,7 @@ public class ServiceBusProcessor : IAsyncDisposable
         /// </value>
         public virtual bool IsProcessing => ActiveReceiveTask != null;
 
-        internal ServiceBusProcessorOptions Options { get; }
+        internal ServiceBusProcessorOptions Options { get; } // TODO contains stale values??
```

Comment:
See also https://github.com/Azure/azure-sdk-for-net/pull/32572/files#r1026946748

Reply:
Yeah this isn't ideal, but it also the values that can get stale are not re-read. We could update to only cache the specific values that we need to reference repeatedly.